src/toast/scripts/toast_benchmark_ground

```diff
@@ -251,6 +265,70 @@ def make_schedule(args, world_comm, site):
     args.schedule = schedule
 
 
+def dump_spt3g(args, job_ops, data):
+    """Save data to SPT3G format."""
+    if not t3g.available:
+        raise RuntimeError("SPT3G is not available, cannot save to that format")
+    save_dir = os.path.join(args.out_dir, "data_spt3g")
+    meta_exporter = t3g.export_obs_meta(
+        noise_models=[
+            (job_ops.default_model.noise_model, job_ops.default_model.noise_model),
+            (job_ops.elevation_model.out_model, job_ops.elevation_model.out_model),
+        ]
+    )
+    # Note that we export detector flags below to a float64 G3TimestreamMap
+    # in order to use FLAC compression.
+    data_exporter = t3g.export_obs_data(
+        shared_names=[
+            (
+                job_ops.sim_ground.boresight_azel,
+                job_ops.sim_ground.boresight_azel,
```

Comment:
This is pretty well documented in the `export_obs_data` class documentation:
https://github.com/hpc4cmb/toast/blob/2b48421ed61f3d1b1334548d28e66201f6b4b72e/src/toast/spt3g/spt3g_export.py#L314-L321
Those tuples define the mapping of names from observation object names to G3 frame object names, and optionally the data type to use on the G3 side.  Since we are just using an arbitrary schema, I am just using the same string for the frame key as the observation name.

I added a comment telling the user to read the class documentation for a description of the constructor arguments.